File: docs/branding/v3.0/11-figma/tokens/GAAhexTokens.swift
import SwiftUI
// GAAhex D18 Color Architecture — one family, one role.
enum GAAhex {
  // COBALT — brand spine / structural chrome
  static let cobalt    = Color(hex: "#1C3B68")
  static let bg        = Color(hex: "#1C3B68")
  static let bgSubtle  = Color(hex: "#16314F")
  static let surface   = Color(hex: "#FFFFFF")
  static let surface2  = Color(hex: "#F4F5F7")
  // GOLD — signature / peak moments only
  static let gold      = Color(hex: "#C5A059")
  static let goldLight = Color(hex: "#AC8847")
  static let goldSoft  = Color(hex: "#EFE3C7")
  // AZURE — interactive only
  static let interactive       = Color(hex: "#0EA5E9")
  static let interactiveHover  = Color(hex: "#0284C7")
  static let interactiveActive = Color(hex: "#0369A1")
  static let interactiveSoft   = Color(hex: "#E0F2FE")
  static let link = interactive, selected = interactive, ring = interactive
  // SLATE — neutrals
  static let text1 = Color(hex: "#0B0B0C"), text2 = Color(hex: "#334155"), text3 = Color(hex: "#64748B")
  static let border = Color(hex: "#E2E5EA"), borderStrong = Color(hex: "#CBD2DA"), divider = Color(hex: "#D8DCE0"), neutral = Color(hex: "#94A3B8")
  // SEMANTIC — status only
  static let success = Color(hex: "#16A34A"), warning = Color(hex: "#D97706"), danger = Color(hex: "#DC2626"), info = Color(hex: "#2563EB")
  static let online = Color(hex: "#16A34A"), provisioned = Color(hex: "#0EA5E9"), qualityGood = Color(hex: "#16A34A"), maintenance = Color(hex: "#D97706")
  static let onColor = Color(hex: "#FFFFFF")
}

File: docs/branding/v3.0/11-figma/tokens/gaahex-tokens.ts
// GAAhex D18 Color Architecture — one family, one role.
export const gaahexTokens = {
  cobalt: { cobalt: "#1C3B68", bg: "#1C3B68", bgSubtle: "#16314F", surface: "#FFFFFF", surface2: "#F4F5F7" },
  gold:   { gold: "#C5A059", goldLight: "#AC8847", goldSoft: "#EFE3C7" },
  azure:  { interactive: "#0EA5E9", interactiveHover: "#0284C7", interactiveActive: "#0369A1", interactiveSoft: "#E0F2FE", link: "#0EA5E9", selected: "#0EA5E9", ring: "#0EA5E9" },
  slate:  { text1: "#0B0B0C", text2: "#334155", text3: "#64748B", border: "#E2E5EA", borderStrong: "#CBD2DA", divider: "#D8DCE0", neutral: "#94A3B8" },
  semantic: { success: "#16A34A", warning: "#D97706", danger: "#DC2626", info: "#2563EB", online: "#16A34A", provisioned: "#0EA5E9", qualityGood: "#16A34A", maintenance: "#D97706", onColor: "#FFFFFF" },
  font: { family: "Sora", word: 500 },
  // @deprecated pre-D18 aliases
  legacy: { cobalt: "#1C3B68", cobaltLift: "#4E7FC4", gold: "#C5A059", ink: "#0B0B0C", cloud: "#F4F5F7", border: "#E2E5EA", silver: "#D8DCE0" },
} as const;
